.github/pull-request-template.md

```diff
@@ -1,40 +1,41 @@
 ## **Description**
-_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
-_1. What is the reason for the change?_
-_2. What is the improvement/solution?_
+
+<!--
+Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
+1. What is the reason for the change?
+2. What is the improvement/solution?
+-->
```

Comment:
Changes the description instructions to a comment so it will not show up in the review and the author can either remove or use it for reference when writing their description.

Reply:
Yes! I have noticed that this has been getting left in commit messages and it creates a lot of noise. Good call.